services/main/Configurations/RedisDataBaseConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NRedisStack;
+using NRedisStack.RedisStackCommands;
+using StackExchange.Redis;
+
+namespace main.Configurations;
+
+public class RedisDataBaseConfiguration
+{
+    public static IDatabase redisDb { get; private set; } = default!;
+
+    public static IDatabase RedisDbConfig()
+    {
+        ConnectionMultiplexer redis = ConnectionMultiplexer.Connect("localhost");
```

Comment:
```suggestion
        ConnectionMultiplexer redis = ConnectionMultiplexer.Connect(envName);
```

We might want to expose this as an env.... so that we can use remote redis when deploying!